src/ResourceManager/Sql/Commands.Sql/ManagedDatabase/Cmdlet/RemoveAzureSqlManagedDatabase.cs

```diff
@@ -0,0 +1,158 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Sql.ManagedDatabase.Model;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Management.Automation;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+
+namespace Microsoft.Azure.Commands.Sql.ManagedDatabase.Cmdlet
+{
+    [Cmdlet(VerbsCommon.Remove, "AzureRmSqlManagedDatabase", SupportsShouldProcess = true)]
+    public class RemoveAzureSqlManagedDatabase : AzureSqlManagedDatabaseCmdletBase<IEnumerable<AzureSqlManagedDatabaseModel>>
+    {
+        protected const string RemoveByNameAndResourceGroupParameterSet =
+            "RemoveManagedDatabaseFromInputParameters";
+
+        protected const string RemoveByInputObjectParameterSet =
+            "RemoveManagedDatabaseFromAzureSqlManagedDatabaseModelInstanceDefinition";
+
+        protected const string RemoveByResourceIdParameterSet =
+            "RemoveManagedDatabaseFromAzureResourceId";
+
+        /// <summary>
+        /// Gets or sets the name of the managed database to remove.
+        /// </summary>
+        [Parameter(ParameterSetName = RemoveByNameAndResourceGroupParameterSet, 
+            Mandatory = true,
+            Position = 0,
+            HelpMessage = "The name of the Azure SQL Managed Database to remove.")]
+        [Alias("Name")]
+        [ValidateNotNullOrEmpty]
+        public string ManagedDatabaseName { get; set; }
```

Comment:
Rename this to Name, and switch order of the parameters to ResourceGroupName, ManagedInstanceName, then Name.